apps/desktop/src/lib/components/AIPromptEdit/AIPromptEdit.svelte

```diff
@@ -51,9 +57,9 @@
 			}}
 		/>
 
-		{#each $userPrompts as prompt}
+		{#each $userPrompts as _prompt, idx}
 			<Content
-				bind:prompt
+				bind:prompt={$userPrompts[idx] as UserPrompt}
```

Comment:
Any idea why the migration script would rewrite it this way?

Reply:
I made this change, it's required in runes mode now apparently because you can't reassign or bind in an each block argument anymore. This is the syntax they recommended